src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/metrics/booleanexpressioncomplexity/Example1.java

```diff
@@ -5,18 +5,20 @@
   </module>
 </module>
 */
+package com.puppycrawl.tools.checkstyle.checks.metrics.booleanexpressioncomplexity;
 
 // xdoc section -- start
-public class Test
+public class Example1
 {
   public static void main(String ... args)
   {
     boolean a = true;
     boolean b = false;
 
-    boolean c = (a & b) | (b ^ a);       // OK, 1(&) + 1(|) + 1(^) = 3 (max allowed 3)
+    boolean c = (a & b) | (b ^ a);       // OK, 1(&)+1(|)+1(^)=3 (max allowed 3)
```

Comment:
I liked the one with spacing, please revert this.

Reply:
Done..
There was this constraint of line length so I had the space removed previously, but I have added new changes
Please review if that is all good.